app/components/ui/molecules/DashboardListItem.js

```diff
@@ -85,12 +85,8 @@ const DashboardListItem = ({ statusCode, title, date }) => (
 
 DashboardListItem.propTypes = {
   statusCode: PropTypes.string.isRequired,
-  title: PropTypes.string,
+  title: PropTypes.string.isRequired,
   date: PropTypes.instanceOf(Date).isRequired,
 }
 
-DashboardListItem.defaultProps = {
-  title: 'Untitled manuscript',
```

Comment:
To be honest, this feels to me like something that the `DashboardListItem` should be handling, rather than the `DashboardList`. What do you think @Cooryd?

How about if we passed the whole manuscript object down from `DashboardList` into `DashboardListItem` & allowed this component to be responsible for rendering appropriately?

Reply:
This makes sense to me, would certainly make more sense to convert the iso date to a Date object within the `DashboardListItem`